vtex-sites/gatsby.store
# feat: Working `SkuSelector` component

## What's the purpose of this pull request?

This PR brings over the implementation made on https://github.com/vtex-sites/nextjs.store/pull/158. This is to make the `SkuSelector` component work.

## How does it work?

This creates a new component called `Selectors`, which renders `SkuSelector`s for each variant property of a certain product. The data that's necessary to render those comes from the [newly added](https://github.com/vtex/faststore/pull/1407) `skuVariants` field from `@faststore/api`.

## How to test it?

Since we didn't have any products with SKU properties configured that would enable the selector to function, in this branch, the deploy preview you'll find is pulling data from the storecomponents account instead of storeframework. So, to see the SKU Selector, go to a product such as [Classic Shoes](https://sfj-63ef1ca--gatsby.preview.vtex.app/classic-shoes-310124175/p).

Here's a video demo: https://www.loom.com/share/8cf41e561ad54b88845e9effca2aeb8c.

## Checklist

<em>You may erase this after checking them all ;)</em>

**PR Title and Commit Messages**

**PR Description**

**Documentation**
